src/CoinbaseWalletSDK.ts

```diff
@@ -68,6 +71,7 @@ export class CoinbaseWalletSDK {
     }
 
     this._overrideIsCoinbaseWallet = options.overrideIsCoinbaseWallet ?? true;
+    this._overrideIsCoinbaseBrowser = options.overrideIsCoinbaseBrowser ?? true;
```

Comment:
why does this have a different default value than what's on CoinbaseWalletProvider? it seems to be this should be false?